File: 2023/php/test/TestDay6B.php
<?php declare(strict_types=1);

$loader = require __DIR__ . '/../vendor/autoload.php';

use day6\b\Solution6B;
use PHPUnit\Framework\TestCase;

final class TestDay6B extends TestCase
{

    public function testExample(): void
    {
        $this->assertEqualsWithDelta(71503, Solution6B::getResult("../resources/6/test_b.txt"), 0);
    }

    public function testResult(): void
    {
        $this->assertEqualsWithDelta(45128024, Solution6B::getResult("../resources/6/input.txt"), 0);
    }

}

File: 2023/java/src/test/java/org/frieder/aoc/day6/TestDay6B.java
package org.frieder.aoc.day6;

import org.frieder.aoc.day6.b.Solution6B;
import org.junit.Test;

import java.io.IOException;

import static org.junit.Assert.assertEquals;

public class TestDay6B {
    @Test
    public void testExample() throws IOException {
        assertEquals(71503, Solution6B.getResult("../resources/6/test_b.txt"),0);
    }

    @Test
    public void testResult() throws IOException {
        assertEquals(45128024, Solution6B.getResult("../resources/6/input.txt"),0);
    }
}
